.github/workflows/build-complete-samples.yml

```diff
@@ -883,7 +883,11 @@ jobs:
             - project_path: 'samples/app-cache-meetings/nodejs'
               name: 'app-cache-meetings'
               version: '18.x'
-             
+
+            - project_path: 'samples/app-anonymous-users/nodejs/ClientApp'
+              name: 'app-anonymous-users'
```

Comment:
Run build pipeline and add screenshots.

Reply:
![image](https://user-images.githubusercontent.com/117716554/221483909-75e57ccf-28f4-4fd2-86fb-e8f85f2f4c5f.png)